pkgs/development/python-modules/pycmarkgfm/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27, cffi, pytest }:
+
+buildPythonPackage rec {
+  pname = "pycmarkgfm";
+  version = "1.0.1";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0wkbbma214f927ikn3cijxsrzkmm5cqz1x4fimrwx9s2wfphj250";
+  };
+
+  propagatedBuildInputs = [ cffi ];
+
+  checkInputs = [ pytest ];
+  checkPhase = ''
+    pytest
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/zopieux/pycmarkgfm";
+    description = "Bindings to GitHub's Flavored Markdown (cmark-gfm), with enhanced support for task lists";
+    license = licenses.gpl3Plus;
```

```suggestion
    license = licenses.gpl3Plus;
    platforms = platforms.all;
```

I am not sure if this is needed here. My current rule is: everyhwhere except Rust and Go.